bfsbdump

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+"""
+Dump BlueField secure boot status information.
+"""
+
+import argparse
+import struct
+import binascii
+import os
+import sys
+import subprocess
+
+LifeCycleState = [ 'Production', 'Secure', 'Non-Secure', 'RMA' ]
+
+def GetPlatformName():
+    Out = subprocess.run(['bfhcafw', 'mcra', '0xf0014.0:16'], stdout=subprocess.PIPE)
+    DeviceId = Out.stdout[:-1].decode('utf-8')
+    if DeviceId == '0x00000211':
+        return 'BlueField1'
+    elif DeviceId == '0x00000214':
+        return 'BlueField2'
+    elif DeviceId == '0x0000021c':
+        return 'BlueField3'
+    else:
+        printf('Error! can not read device id')
```

Comment:
Why is this `printf`?

In addition, errors should be printed through stderr, which can be achieved with
```python
print("message", file=sys.stderr)
```

Reply:
Oops. my bad.. too much C (I guess)